analyzer/codechecker_analyzer/analyzer.py

```diff
@@ -275,15 +274,20 @@ def perform_analysis(args, skip_handler, context, actions, metadata_tool,
         if ((ctu_collect and not ctu_analyze)
                 or ("stats_output" in args and args.stats_output)):
             pre_anal_skip_handler = skip_handler
-
-        pre_analysis_manager.run_pre_analysis(pre_analyze,
-                                              context,
-                                              config_map,
-                                              args.jobs,
-                                              pre_anal_skip_handler,
-                                              ctu_data,
-                                              statistics_data,
-                                              manager)
+        clangsa_config = config_map.get(ClangSA.ANALYZER_NAME, None)
```

Comment:
By default [get](https://docs.python.org/3/library/stdtypes.html#dict.get) will return with None if the key is not found in the map:

```suggestion
        clangsa_config = config_map.get(ClangSA.ANALYZER_NAME)
```